inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -827,3 +830,88 @@ def extract_authors_from_pdf(obj, eng):
                 grobid_authors_count
             )
     return obj
+
+
+def prepare_collaboration_multi_search(collaborations):
+    compiled_regexp = get_collaboration_cleaner_regexp()
+    multi_search = MultiSearch(index="records-experiments", using=current_search_client)
+    backup_multi_search = MultiSearch(index="records-experiments", using=current_search_client)
+    for collaboration in collaborations:
+        full_collaboration_string = collaboration.get('value', '')
+        normalized_collaboration_string = compiled_regexp.sub('', full_collaboration_string).lower().strip()
```

Comment:
why do you need to lower if that's already done by ES through analyzer/normalizer?

Reply:
Leftover. Thanks for spotting this. :+1: